demisto_sdk/commands/common/tools.py

```diff
@@ -1105,6 +1107,14 @@ def find_type(path: str = '', _dict=None, file_type: Optional[str] = None, ignor
                     return FileType.INCIDENT_FIELD
                 if _id.startswith('indicator'):
                     return FileType.INDICATOR_FIELD
+                else:  # not an incident or an indicator
+                    if 'definitionId' in _dict and 'genericModuleId' in _dict:
```

Comment:
We might not save the `genericModuleId` field on content repo. so let's remove this condition (same for incident types above)